modules/abstract-utxo/src/abstractUtxoCoin.ts

```diff
@@ -720,6 +735,13 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
     params: VerifyTransactionOptions<TNumber>
   ): Promise<boolean> {
     const { txParams, txPrebuild, wallet, verification = { allowPaygoOutput: true }, reqId } = params;
+
+    // TODO: BG-77743 Add native psbt support
+    if (txPrebuild.txHex && bitgo.isPsbt(txPrebuild.txHex)) {
+      const psbt = bitgo.createPsbtFromHex(txPrebuild.txHex, this.network);
+      txPrebuild.txHex = psbt.getUnsignedTx().toHex();
+    }
```

Comment:
can we do this inside `parseTransaction` instead?

Reply:
`verifyTransanction` is also called from wallet.ts (common to coins) too in our flow where we don't want this.